data/json/vehicle_parts.json

```diff
@@ -5334,5 +5334,24 @@
         "breaks_into" : [
             {"item": "plastic_chunk", "min": 1, "max": 2}
         ]
+    },
+    {
+        "type" : "vehicle_part",
+        "id" : "vehicle_scoop",
+        "name" : "scoop",
+        "symbol" : "R",
+        "color" : "light_gray",
+        "broken_symbol" : ";",
+        "broken_color" : "dark_gray",
+        "durability" : 20,
+        "epower" : -5,
```

Comment:
Yeah, it was meant to be a placeholder before I got around to figuring out the cost of running the thing.